src/components/Form.jsx

```diff
@@ -1,50 +1,47 @@
-import React, { useState } from 'react'
-import { useDispatch } from 'react-redux'
-import { addBook } from '../redux/books/books'
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃